src/types.ts

```diff
@@ -1115,5 +1115,56 @@ export interface SearchReponse {
 }
 
 export interface TransactionResult {
-  txHash: string
-}
\ No newline at end of file
+  txHash: string;
+}
+
+export interface ChatConfigureParamsBase {
+  objectId: string;
+}
+
+export interface OpenAIChatConfigureParams extends ChatConfigureParamsBase {
+  provider: 'openai';
+  api: 'assistants';
+}
+
+export type ChatConfigureParams = OpenAIChatConfigureParams;
+
+export interface AssistantCreateParamsBase {
+  tokenId: string;
+  name: string;
+  instructions: string;
+  description?: string | null;
+}
+
+export interface OpenAIAssistantCreateParams
```

Comment:
(suggestion) ditto, can this be just merged into AssistantCreateParams?

Reply:
Just like the previous case, I divided then because different providers might support different models.
I'll follow the same method we decide on above!

Reply:
That makes sense. Thanks!

Reply:
Thank you for comment on this code too!